// @didi/chameleon-ui-builtin/components/button/type.js
export const STYLE_MAP = {
  red: {
    'background-color': '#db2a36'
  },
  orange: {
    'background-color': '#fc9153'
  },
  white: {
    'background-color': '#FFFFFF',
    'border-color': '#A5A5A5',
    'border-width': '1px'
  },
  blue: {
    'background-color': '#4a4c5b'
  },
  green: {
    'background-color': '#50a050'
  }
};

export const TEXT_STYLE_MAP = {
  red: {
    color: '#FFFFFF'
  },
  orange: {
    color: '#FFFFFF'
  },
  blue: {
    color: '#FFFFFF'
  },
  white: {
    color: '#3D3D3D'
  },
  green: {
    color: '#FFFFFF'
  }
};

export const BUTTON_STYLE_MAP = {
  full: {
    width: '702px',
    height: '88px'
  },
  big: {
    width: '339px',
    height: '70px'
  },
  medium: {
    width: '218px',
    height: '60px'
  },
  small: {
    width: '157px',
    height: '44px'
  }
}

export const TEXT_FONTSIZE_STYLE_MAP = {
  full: {
    'font-size': '36px'
  },
  big: {
    'font-size': '32px'
  },
  medium: {
    'font-size': '28px'
  },
  small: {
    'font-size': '24px'
  }
};
